shell/browser/electron_download_manager_delegate.cc

```diff
@@ -130,6 +245,29 @@ void ElectronDownloadManagerDelegate::OnDownloadPathGenerated(
     const bool offscreen = !web_preferences || web_preferences->IsOffscreen();
     settings.force_detached = offscreen;
 
+#if BUILDFLAG(IS_WIN)
+    auto extension = settings.default_path.FinalExtension();
+    if (!extension.empty() && settings.filters.empty()) {
+      extension.erase(extension.begin());
```

Comment:
This seems unnecessary since the `.` is added back later in `file_ext{u"." + ext}`.